### DISCUSSION TITLE
简单问题——No connection could be made because the target machine actively refused it .

### DISCUSSION THREAD

**Discussion by Noob-in-computers** [Mrs4s/go-cqhttp#1506] [Q&A]
![image](https://user-images.githubusercontent.com/105729522/169480056-3558e7b9-0612-42d6-bd70-993e77e1dafe.png)
买了个服务器，在服务器上部署，但是第一步就出了差错，请问该怎么做，谢谢各位了。

**Comment by Noob-in-computers** [comment]
额……自己鼓捣鼓捣弄好了……原理是我没开bot（捂脸
话说谁知道怎么把这玩意close啊（悲

**Reply by cgx123111** [reply]
什么是bot没开啊,运行go-cqhttp.bat登录进去还要干啥?现在机器人只能发不能交互,我是小白（捂脸

**Reply by bywhite0** [reply]
> 什么是bot没开啊,运行go-cqhttp.bat登录进去还要干啥?现在机器人只能发不能交互,我是小白（捂脸

他指的是他gocq用反向ws连接的bot程序没开所以gocq才会报无法连接
